automations/credits.mdx

```diff
@@ -0,0 +1,38 @@
+---
+title: "Automation Credits"
+description: "Understand how Automation Credits are counted."
+---
+
+The Koala Automations product is available on all plans.
+The Business and Pro plans include unlimited action credits, while the other plans include
+limited credits.
+
+
+## Understanding Credits
+Each succesfull Automation run deducts 1 credit from your available credit pool.
+
+There are separate pools for Actions, Auto Outbound and Slack Alerts, so you can try
+all three products individually on your own time.
+
+Credits are deducted when any succesfull Automation is run. 
```

Comment:
Is it when the run at all? Or when they deliver successfully?

Reply:
There's a difference in how these count for each product though:
- auto outbound = *autopilot* syncs the prospect to your destination sequence/campaign
- actions = successful delivery?
- slack alerts = successful delivery?